internal/lib/lib.go

```diff
@@ -28,20 +29,17 @@ func DumpEmbeddedWAF() (path string, err error) {
 	path = file.Name()
 
 	defer func() {
+		allErrors := []error{err}
```

Comment:
IIRC, you could just `err = errors.Join(err, newErr)` and spare the slice allocation?

Reply:
Done